### PR TITLE
NCDA - Aggregated Scorecard: Universal Intervention pane - ECD

### PR DESCRIPTION
**Pull request by anvmn**
#729

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1896

File: server/elm/src/Backend/Scoreboard/Model.elm [hunk 1]
```diff
@@ -145,6 +137,104 @@ type VaccineDose
...
+    | FollowSimpleInstructions
+    | KickBall
+    | PointAtNamedObjects
+      -- From 3 years.
+    | DressThemselves
+    | WashHandsGoToToiled
+    | KnowsColorsAndNumbers
+    | UseMediumPhrases
+    | PlayMakeBelieve
+      -- From 4 years.
```
Comment (AronNovak) [inline server/elm/src/Backend/Scoreboard/Model.elm:198]: Nicely commented, then it's a baby development guide for coders :)

File: server/hedley/modules/custom/hedley_chw/hedley_chw.module [hunk 2]
```diff
@@ -288,12 +288,12 @@ function hedley_chw_participate_in_group($child_id, $adult_id, $group_id) {
  * @throws \EntityMetadataWrapperException
  */
 function hedley_chw_get_village_residents($nid, $range = 2000) {
-  $wrapper = entity_metadata_wrapper('node', $nid);
-  $province = $wrapper->field_province->value();
-  $district = $wrapper->field_district->value();
-  $sector = $wrapper->field_sector->value();
-  $cell = $wrapper->field_cell->value();
-  $village = $wrapper->field_village->value();
+  $node = node_load($nid);
```
Comment (AronNovak) [inline server/hedley/modules/custom/hedley_chw/hedley_chw.module:291]: what was the issue with EntityMetadataWrapper?

Comment (anvmn) [reply server/hedley/modules/custom/hedley_chw/hedley_chw.module:291]: @AronNovak  Memory leakage.
When executed from Drush script, every time this function was invoked, we'd get +2M of RAM.
After ~120 iterations, we run out of memory. 
Tried `drupal_static_reset()` and some other stuff, like `unset()`, but it made no difference.
Maybe you got other idea?

Comment (AronNovak) [reply server/hedley/modules/custom/hedley_chw/hedley_chw.module:291]: Nah, let's leave it like that.

**Review by AronNovak — APPROVED** [review]

**Pull request merged**